weather_api/main.js

```diff
@@ -0,0 +1,72 @@
+//95137d96d3dd21f446c9da25b57a24be
+//http://api.weatherstack.com/
+// let weatherCurrentUrl = "http://api.weatherstack.com/current"
+let access_key = "95137d96d3dd21f446c9da25b57a24be";
+let chosenCity = document.getElementById("city");
+
+
+// let search = document.getElementById("search_city");
+let w_icon = document.getElementById("w_icon");//картинка
```

Comment:
Please, don't use underscores in variable names! It's a bad practice and is not suggested in popular style guides (e.x. [Airbnb](https://airbnb.io/javascript/#naming--camelCase)) and other javascript conventions ( read more [here](https://www.crockford.com/code.html) or [here](https://www.freecodecamp.org/news/javascript-naming-conventions-dos-and-don-ts-99c0e2fdd78a/))